### PR TITLE
set num levels

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
add set num levels api

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@BusyJay @siddontang @ngaut PTAL

File: src/rocksdb_options.rs [hunk 1]
```diff
@@ -563,6 +563,12 @@ impl Options {
                                                       period);
         }
     }
+
+    pub fn set_num_levels(&mut self, n: c_int) {
```
Comment (BusyJay) [inline src/rocksdb_options.rs]: Any tests?

**Comment by siddontang** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**